cloud/testserver/cloudstore/store.go

```diff
@@ -208,15 +208,15 @@ outer:
 }
 
 // GetStackByMetaID returns the given stack.
-func (d *Data) GetStackByMetaID(org Org, id string) (*Stack, int, bool) {
+func (d *Data) GetStackByMetaID(org Org, id string) (Stack, int, bool) {
 	d.mu.RLock()
 	defer d.mu.RUnlock()
 	for i, st := range org.Stacks {
 		if st.Stack.MetaID == id {
-			return &st, i, true
+			return st, i, true
```

Comment:
as we are returning here, it wasn't actually buggy because the `st` scoped variable was escaping with the last value (the right one). But it's indeed a bad practice getting the pointer of a range variable, then it was best to just use a value instead of a pointer.
Thanks @wmalik for pointing this out.